app/commands/user/reputation_token/award_for_pull_request_reviewers.rb

```diff
@@ -14,7 +14,12 @@ def call
           map { |reviewer| reviewer[:reviewer_username] }.
           compact.
           uniq
-        reviewer_usernames.delete(params[:author_username]) # Don't award reviewer reputation to the PR author
+
+        # Don't award reviewer reputation to the PR author
+        reviewer_usernames.delete(params[:author_username])
+
+        # Only award reviewer reputation to organization members
+        reviewer_usernames &= ::Github::OrganizationMember.pluck(:username)
```

Comment:
```suggestion
        reviewer_usernames += ::Github::OrganizationMember.pluck(:username)
```

(I've never seen `&=` before)

Reply:
That's not the same I think. `&=` was introduced in Ruby 2.6 and does an intersection.

Reply:
Oh I see. That's weird but cool.